### PR TITLE
PIMS-1653: Adding all the layers to the map

### PR DESCRIPTION
**Pull request by Sharala-Perumal**
PIMS-1653:  Adding all the layers to the map                                  

## 🎯 Summary

  
[PIMS-1653](https://citz-imb.atlassian.net/browse/PIMS-1653)

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.

### CODE REVIEW

**Comment by GrahamS-Quartech** [comment]
I toggled through all the different layers and they all appear to be working correctly as far as I can tell. The layer control should be positioned differently though, it's completely hidden when the property sidebar is open (which is the default). I also think that the parcel layer should probably be enabled by default (since I think this is the one that people actually use most of the time?)

**Comment by Sharala-Perumal** [comment]
> I toggled through all the different layers and they all appear to be working correctly as far as I can tell. The layer control should be positioned differently though, it's completely hidden when the property sidebar is open (which is the default). I also think that the parcel layer should probably be enabled by default (since I think this is the one that people actually use most of the time?)

Good suggestions, Graham. Map should now render Parcel Boundaries as default layer and the layers control should display on the top left corner.

File: react-app/src/components/map/MapLayers.tsx [hunk 1]
```diff
@@ -1,25 +1,181 @@
...
-        url={PARCEL_LAYER_URL}
-        format="image/png; mode=8bit"
-        transparent={true}
-        opacity={0.5}
-        layers="WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_SVW"
-      />
-    </>
-  );
-};
+    <LayersControl.Overlay name="Administrative Boundaries">
```
Comment (GrahamS-Quartech) [inline react-app/src/components/map/MapLayers.tsx:99]: I can see the idea behind setting up these parent components and having the enumerated children nested with a bit of left margin to make them look grouped together but I don't think it's working. It seems like the control displays identically when these parent LayersControl.Overlay's are removed. There are some leaflet plugins out there that provide this kind of functionality if it's really needed.

**Comment by GrahamS-Quartech** [comment]
> Good suggestions, @GrahamS-Quartech . Map should now render Parcel Boundaries as default layer and the layers control should display on the top left corner.

These changes look good.

**Review by dbarkowsky — APPROVED** [review]

**Review by GrahamS-Quartech — APPROVED** [review]

**Pull request merged**